base/promotion.jl

```diff
@@ -217,13 +217,27 @@ max(x::Real, y::Real) = max(promote(x,y)...)
 min(x::Real, y::Real) = min(promote(x,y)...)
 minmax(x::Real, y::Real) = minmax(promote(x, y)...)
 
-# "Promotion" that takes a function into account. You can override this
-# as needed. For example, if you need to provide a custom result type
-# for the multiplication of two types,
-#   promote_op{R<:MyType,S<:MyType}(::typeof(*), ::Type{R}, ::Type{S}) = MyType{multype(R,S)}
+# "Promotion" that takes a function into account. These are meant to be used
+# mainly by broadcast methods, so it is adviced against overriding them
```

Comment:
If the CI queue is long and there are no more code changes to make then that can help, yeah (it can be squashed on merge). The queue is quiet right now though so doesn't make much difference.